.github/workflows/manual_gcs_release_test.yml

```diff
@@ -0,0 +1,17 @@
+name: Test GCS Release Bucket
```

Comment:
This workflow utilizes the same auth steps as the release workflow, giving us confidence that the release workflow will be successful when uploading to GCS.